src/components/Help/Documentation/index.tsx

```diff
@@ -0,0 +1,219 @@
+// @flow
+import React from 'react';
+
+import config from 'config';
+import Link from 'components/generic/Link';
+import loadable from 'higherOrder/loadable';
+import { schemaProcessDataPageSection } from 'schema_org/processors';
+
+import fonts from 'EBI-Icon-fonts/fonts.css';
+import ipro from 'styles/interpro-new.css';
+import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
+import local from './style.css';
+
+import cssBinder from 'styles/cssBinder';
+const f = cssBinder(ebiGlobalStyles, fonts, ipro, local);
+
+const SchemaOrgData = loadable({
+  loader: () => import(/* webpackChunkName: "schemaOrg" */ 'schema_org'),
+  loading: () => null,
+});
+
+const Documentation = () => {
+  return (
+    <section>
+      <SchemaOrgData
+        data={{
+          name: 'InterPro Documentation Page',
+          description:
+            'Includes links to the documentations for the different parts of our service',
+        }}
+        processData={schemaProcessDataPageSection}
+      />
+      <div className={f('row')}>
```

Comment:
`row` and `columns` are foundation styles I'm trying to move away, try to use `vf-stack` instead